test.js

```diff
@@ -51,6 +51,22 @@ Ponies|
 	t.is(actual, expected);
 });
 
+test('correctly encodes quotes in the `href` option', t => {
+	const actual = bitbar.create([
+		{
+			text: 'Unicorns',
+			color: '#ff79d7',
+			href: 'https://www.youtube.com/watch?v=\'9auOCbH5Ns4&time=2',
```

Comment:
#11 also mentions ampersands, can you include that in the test here too? And also test manually with BitBar? I would also include double-quotes, just to be sure.